news/2018-04-08-project-loved-week-of-april-8th.md

```diff
@@ -0,0 +1,279 @@
+
+[Five Hammer - f f f f f](https://osu.ppy.sh/beatmapsets/436455) by [Vincs](https://osu.ppy.sh/users/2026299)  
+*written by [Tenshichan](https://osu.ppy.sh/users/1101600)*
+
+f f f f f mapped by [Vincs](https://osu.ppy.sh/users/2026299) is yet another CS0 map going for the loved category. What seperates this one from the other ones is that this one has more of a focus on stamina than technical patterns. Although it is only roughly 2 minutes long, due to the vast amount of fast left-right dashing it is not that easy to keep consistent. All in all the map serves as a nice stamina practice especially for aspiring players who always seek new challenges.
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/catch/sink-to-the-deep-sea-world.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Chroma - sink to the deep sea world](https://osu.ppy.sh/beatmapsets/650973) by [\_Asriel](https://osu.ppy.sh/users/566276)  
+*written by [SartNYAN OwO xD](https://osu.ppy.sh/users/4100941)*
+
+sink to the deep sea world by [\_Asriel](https://osu.ppy.sh/users/566276) is a fairly new, but also very well acclaimed map. For this one you are definitely in need of fast fingers and great control over anti-flow type of patterns. [\_Asriel](https://osu.ppy.sh/users/566276) has become known for making some of the most challenging osu!catch maps in recent times, with some of his maps being quite clunky and questionable playability wise. This map of his, however, has been modded and alternated with multiple times and has become a very polished product of [\_Asriel](https://osu.ppy.sh/users/566276)'s. Anyhow, go have a shot at this one, break a leg, and then return over here to make your voting decision.
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/catch/apparition.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Nanahoshi Kangengakudan - IMAGINARY LIKE THE JUSTICE](https://osu.ppy.sh/beatmapsets/507020) by [-Ken](https://osu.ppy.sh/users/4430811)  
+*written by [SartNYAN OwO xD](https://osu.ppy.sh/users/4100941)*
+
+Apparition from [Razor Sharp](https://osu.ppy.sh/users/3414261) is definitely one hell of a map. This super strenuous, fast pased, hyper-heavy map is a very neat challenge for top of the line players. It is also not your typical song you would expect from osu!, it being technological death metal. Apparition has seen plenty of play in multiplayer lobbies in the past, showing just how well known this underappreciated map from [Razor Sharp](https://osu.ppy.sh/users/3414261) has become through the course of time. So, if you are up for the challenge and believe you can have a shot at this behemoth, go and vote!
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/catch/imaginary-like-the-justice.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Spawn Of Possession - Apparition](https://osu.ppy.sh/beatmapsets/425956) by [Razor Sharp](https://osu.ppy.sh/users/3414261)  
```

Comment:
This line and line 178 are reversed (or something more complicated than that happened).